_posts/2021-06-03-opensearch-release-candidate-announcement.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "The OpenSearch Release Candidate is now available on GitHub"
```

Comment:
Let's call this "OpenSearch Release Candidate (RC1) is now available". RC1 will be aligned to the framing on the roadmap in GitHub.

Reply:
Updated!